dockertest/subtestbase_unittests.py

```diff
@@ -72,6 +74,36 @@ class DockerTestFail(Exception):
 ]
 
 
+class TestNotRedHat(TestCase):
+
+    def setUp(self):
+        import subtestbase
+        self.subtestbase = subtestbase
+        # Saves some typing
+        self.stbsb = self.subtestbase.SubBase
+        pfx = 'subtestbase_unitest'
+        (self.fd,
```

Comment:
Ahh sure, but it's not very faithful to the unittesting spirit if "peeking under the covers" is allowed :grinning: